src/controller/config/controller.go

```diff
@@ -173,15 +173,36 @@ func (c *controller) ConvertForGet(ctx context.Context, cfg map[string]interface
 	if err != nil {
 		return nil, err
 	}
-	result[common.AUTHMode].Editable = flag
+	result[common.AUTHMode].Editable = flag && !readOnlyForAll
 
 	return result, nil
 }
 
+func (c *controller) OverwriteConfig(ctx context.Context) error {
+	mgr := config.GetCfgManager(ctx)
+	cfgMap := map[string]interface{}{}
+	if v, ok := os.LookupEnv(configOverwriteJSON); ok {
+		err := json.Unmarshal([]byte(v), &cfgMap)
+		if err != nil {
+			return err
+		}
+		err = mgr.ValidateCfg(ctx, cfgMap)
```

Comment:
The `mgr.ValidateCfg` still called twice? It's called again in `UpdateUserConfigs`.

Reply:
Updated